setup.py

```diff
@@ -4,7 +4,7 @@
     "blspy==1.0.4",  # Signature library
     "chiavdf==1.0.2",  # timelord and vdf verification
     "chiabip158==1.0",  # bip158-style wallet filters
-    "chiapos==1.0.3",  # proof of space
+    "chiapos==1.0.4",  # proof of space
```

Comment:
We can tag the chiapos branch as `1.0.4b0` and mark it as a pre release. This way we can test this PR before merging it.